sigstore-public-deployment-spec.md

```diff
@@ -111,12 +114,14 @@ primary deviations are described in [Spec: Rekor](./rekor-spec.md); however,
 like RFC 6962, the transparency service specification leaves specific parameter
 choices to implementers.
 
-* Base URL: [https://rekor.sigstore.dev/](https://rekor.sigstore.dev/)
+* Base URL: defined in [Root-of-Trust](#distributing-roots-of-trust)
+* Public Key: defined in [Root-of-Trust](#distributing-roots-of-trust)
 * Hash Algorithm: SHA-256 ([RFC 6234](https://datatracker.ietf.org/doc/rfc6234/); OID 2.16.840.1.101.3.4.2.1)
 * Signature Algorithm:  ECDSA (NIST P-256).
-* Public Key: See [root-signing repo](https://github.com/sigstore/root-signing)
 * Log ID: the SHA-256 hash of the log's public key
-* Maximum Merge Delay: Rekor only returns an index after the merge is complete. Rekor does not support batching. Rekor returns an inclusion proof after waiting for an entry to be included in the log, which is expected to take <1s.
+* Inclusion and merge delays:
+  * Rekor v2 will always return an entry with inclusion proof: This introduces a delay that depends on service configuration
+  * Rekor v1 may return an entry with inclusion promise (a Signed Entry Timestamp) only
 * Identity monitoring: [Rekor monitor](https://github.com/sigstore/rekor-monitor)
 
 ### 3.3 Sharding
```

Comment:
Maybe we should mention that sharding in rekor v2 is a more integral part of the operation and link to the rekor v2 spec?